### DISCUSSION TITLE
dosxyz_show Ubuntu does not work

### DISCUSSION THREAD

**Discussion by Mariag-hub** [nrc-cnrc/EGSnrc#1085] [Questions] [answered]
Hi everyone!
I'm new to the esgnrc world and I have some problems displaying the dose map on my phantom. I successfully did a simulation and I've seen the results using Victoria program online.
When I try to open the same phantom and 3d dose map using the dosexyz_show app it goves me an error

>  gray shades: 128
>  128 gray shades allocated!
> X Error of failed request:  BadName (named color or font does not exist)
>   Major opcode of failed request:  45 (X_OpenFont)
>   Serial number of failed request:  744
>   Current serial number in output stream:  749

Does anyone know how to solve this? I'm using Ubuntu, and I know that this tool only works on Linux distributions, so I thought it would have worked just fine.

Thank you in advance.

**Comment by rtownson** [comment]
It looks like a font package is missing, which version of ubuntu are you using? Try installing the following one at a time, and let me know which one fixes it:

```
xorg-font-util
xorg-fonts-100dpi
xorg-fonts-75dpi
xorg-fonts-alias
xorg-fonts-encodings
xorg-fonts-misc
xorg-fonts-type1
xorg-mkfontscale
```

**Reply by Mariag-hub** [reply]
I'm using Ubuntu 22.04.3 LTS. When I try installing one of these packages it gives me the same error:
`E: Unable to locate package xorg-fonts-alias`
I tried also updating apt-get install but nothing works for now

**Reply by rtownson** [reply, marked as answer]
Sorry that was for Fedora, my mistake. Try these. If they don't work, you may have to search for other font packages.

```
xfonts-base
xfonts-100dpi
xfonts-75dpi
xfonts-cyrillic
```

By the way, it's also safe to just use `VICTORIA` instead, `dosxyz_show` is not superior.

**Reply by Mariag-hub** [reply]
Thanks, I had to install both `xfonts-base` and `xfonts-cyrillic` to make it work. Thanks for your help
